tests/test_sensors.py

```diff
@@ -10,101 +10,106 @@
 from rocketpy.sensors.gyroscope import Gyroscope
 
 
-def test_sensor_on_rocket(calisto_sensors):
+def test_sensor_on_rocket(calisto_with_sensors):
```

Comment:
This file name should be `test_sensor` in accordance with the `test_flight` (for example) we already have.

Same comment apply to the `test_sensors` inside the `unit` folder.